org.kde.kfind.json

```diff
@@ -0,0 +1,22 @@
+{
+    "id": "org.kde.kfind",
+    "branch": "master",
+    "rename-icon": "kfind",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kfind",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland", "--filesystem=host" ],
+
+    "modules": [
+        {
+            "name": "kfind",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                { "type": "git",
+                  "commit": "260fec641fa8a596144e8bb3cbfbab92fb9034df",
```

Comment:
you need to use a tarball not a git commit

Reply:
Thanks, the latest commit now uses tarball.